js/client-teambuilder.js

```diff
@@ -463,11 +463,10 @@
 			}
 		},
 		defaultDragEnterTeam: function (e) {
-			// Drag originated outside teambuilder
-			if (!e.originalEvent.dataTransfer) return;
-			if (!e.originalEvent.dataTransfer.files) return;
-			if (e.originalEvent.dataTransfer.files[0] && e.originalEvent.dataTransfer.files[0].name.slice(-4) !== '.txt') return;
```

Comment:
I do have a Linux version, but whenever I ask for someone to test it, zero people respond. Can you test it?

http://pokemonshowdown.com/files/pokemonshowdown-linux-x64.tar.gz